terraform/accounts/main/s3_buckets.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_s3_bucket" "database_backups_s3_bucket" {
+  bucket = "digitalmarketplace-database-backups"
+  acl    = "private"
+
+  versioning {
+    enabled = true
```

Comment:
If terraform complains in the future we'll need to add `mfa_delete = true` to this once we've switched it on.